scripts/bootstrap

```diff
@@ -142,6 +142,10 @@ then
   npm install "${tgz_file}" --prefix "${static_folder}"
 fi
 
+# install cypress
+info_msg "Install cypress"
+npm install --prefix "tests/e2e/cypress"
```

Comment:
Does it mean that it will install all previous installed npm dependencies in another directory?

Is that possible to configure cypress to use rero-ils npm dependencies? Or to just install cypress in `${static_folder}`?

Reply:
Cypress is a testing application and I think its place is not in static folder but in tests folder. It will not install `all`previous installed npm dependencies, but only the default ones and those specified in corresponding package.json.

Reply:
Cypress dependencies takes 53Mb on my side. RERO dependencies takes 150Mb.
What about this place in our Dockerfile? It will grow up the image, but without any improvement on RERO-ils functionnality.

I suggest to add an option to avoid installing these dependencies.

Reply:
@blankoworld Cypress should not be installed into a Docker image at all. It is just a dev tool. For me it should be installed in his own directory and should be installed on demand. For example by adding an option to the bootstrap or with his own script.

Reply:
I agree with the option in bootstrap or an own script.

Reply:
I agree that Cypress should not be in Docker image. But with the given script in bootstrap, it is. And that was my question: why not excluding it from the Docker image (and so, we should find a solution).

The solution you suggest is a good idea because cypress will either not be in docker image nor in bootstrap/setup.

Reply:
The solution we choose to use is a separated script